### ISSUE TITLE
Meaningful accessibility label missing on contacts

### ISSUE THREAD

**Issue by jleach** [bcgov/bc-wallet-mobile#2015]
All contacts in table have the same "a contact" accessibility label making them difficult.  Found in 1.0.17 1744

Expected Behaviour
- Accessibility label should match the name of the contact.

![Image](https://github.com/bcgov/bc-wallet-mobile/assets/390891/694a312d-d372-4b45-a595-6e19b1bcec7f)

**Comment by jeznorth** [comment]
Verified complete (IOS 18.4.1 / Android 15).
